tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -225,9 +223,11 @@ Matcher<std::vector<float>> ArrayFloatNear(const std::vector<float>& values,
   matchers.reserve(values.size());
   for (const float& v : values) {
     if (nan_sensitive) {
-      matchers.emplace_back(NanSensitiveFloatNear(v, max_abs_error));
+      matchers.emplace_back(::testing::NanSensitiveFloatNear(v, max_abs_error));
+    } else if (max_abs_error > 0) {
```

Comment:
If we will keep the three branch, I would either make this routine assert max_abs_error >= 0 or rewrite the if-stmt as follows so that ::testing::FloatNear will issue an error when max_abs_error is a negative value:
if (nan_sensitive) {
...
} else if (max_abs_error == 0) {
...
} else {
...
}

Reply:
Added a check for negative values of max_abs_error.